js/works.js

```diff
@@ -0,0 +1,107 @@
+// let reverse;
+
+// const cardsinfo = [
+//   {
+//     titulo: 'tonic',
+//     subtitulo: 'CANOPY',
+//     description: 'asdfasdfasd',
+//   },
+// ];
+
+// const card = `<article class="work ">
+// <img
+//   class="work-main-img"
+//   alt="image of the first proyect"
+//   src="./assets/Snapshoot Portfolio.svg"
+//   alt=""
+// />
+// <aside class="work-info">
+//   <h2 class="work-title">Tonic</h2>
+//   <ul class="work-list">
+//     <h4> ${cardsinfo[0].subtitulo} </h4>
+//     <li class="work-list-item">
+//       <img src="./assets/Counter.svg" alt="" />
+//       Back End Dev
+//     </li>
+//     <li class="work-list-item">
+//       <img src="./assets/Counter.svg" alt="" />
+//       2015
+//     </li>
+//   </ul>
+//   <article class="work-description">
+//     <p>
+//       ${cardinfo.description}
+//     </p>
+//   </article>
+//   <ul class="work-tech">
+//     <li>html</li>
+//     <li>css</li>
+//     <li>javaScript</li>
+//   </ul>
+//   <article class="work-button">
+//     <button type="button" class="btn">See project</button>
+//   </article>
+// </aside>
+// </article>`;
+
+// const container = document.querySelector('.works-container');
+
+// const div = document.createElement('div');
+
+// div.innerHTML = card;
+
+// container.appendChild();
+
+// console.log(card);
+
+// const fragment = document.createDocumentFragment();
+
+// const worksContainer = document.querySelector('.works-container');
+
+// export function createCard(reverse) {
+//   const card = `<article class="work work-${reverse}">
+// <img
+//   class="work-main-img"
+//   alt="image of the first proyect"
+//   src="./assets/Snapshoot Portfolio.svg"
+//   alt=""
+// />
+// <aside class="work-info">
+//   <h2 class="work-title">Tonic</h2>
+//   <ul class="work-list">
+//     <h4>CANOPY</h4>
+//     <li class="work-list-item">
+//       <img src="./assets/Counter.svg" alt="" />
+//       Back End Dev
+//     </li>
+//     <li class="work-list-item">
+//       <img src="./assets/Counter.svg" alt="" />
+//       2015
+//     </li>
+//   </ul>
+//   <article class="work-description">
+//     <p>
+//       A daily selection of privately personalized reads; no accounts
+//       or sign-ups required.
+//     </p>
+//   </article>
+//   <ul class="work-tech">
+//     <li>html</li>
+//     <li>css</li>
+//     <li>javaScript</li>
+//   </ul>
+//   <article class="work-button">
+//     <button type="button" class="btn">See project</button>
+//   </article>
+// </aside>
+// </article>`;
+//   const fragment = document.createDocumentFragment();
+//   const worksContainer = document.querySelector('.works-container');
+
+//   fragment.append(card);
+// }
+
+// const card1 = {
+//   src: './assets/Snapshoot Portfolio.svg',
+//   titulo: 'topic',
+// };
```

Comment:
- Please, let's remove these commented lines of code to make our codebase clean.
- Also, since you're not making use of this file, you can as well remove it from your repo.